apps/dashboard/package.json

```diff
@@ -33,7 +33,7 @@
     "@bdelab/roar-utils": "^1.2.1",
     "@bdelab/roar-vocab": "1.8.0",
     "@bdelab/roav-crowding": "1.1.35",
-    "@bdelab/roav-mep": "1.2.2",
+    "@bdelab/roav-mep": "^1.2.6",
```

Comment:
I'd like to use explicit version numbers for the assessments rather than the caret operator.